### PR TITLE
Website instructions

### LINKED ISSUE #32: Add instructions for adding a heroku db on the website
I think it's fairly likely that not all the students will have the heroku CLI as it seems like more and more people are using the heroku interface online to connect to github repos and auto-deploy etc.

As such this MC needs either a requirement for the heroku CLI at the top and mentors need to make sure everyone has this installed beforehand (as an hour is already short for this challenge and probably shouldn't waste time installing stuff) **or**(and?) we add instructions for adding a DB through the heroku website.

### PR DESCRIPTION
**Pull request by m4v15**
fixes #32 

adds content section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,15 +42,24 @@ The purpose of this challenge is to construct complicated queries, learn about j
   `$ echo "export DATABASE_URL = {YOUR_COPIED_DATABASE_URL}" >> "config.env"`
 - Build your database by running: `$ node database/db_build.js`
 
+#### Using the website
+- Fork this repo into your own github account.
+- On the heroku dashboard create a new app, giving it a unique name
```
[the remark this answers is not in the document]
Comment (m4v15) [reply README.md]: oh yeah I had moved it out before but changed it back

**Comment by shiryz** [comment]
@m4v15 conflicts :disappointed:

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (shiryz) [reply README.md]: So you want the first 3 instructions to be separated?

**Comment by m4v15** [comment]
@shiryz fixed

**Comment by m4v15** [comment]
@eliascodes you think it's good to merge - nothing fundemental has changed

**Comment by eliascodes** [comment]
@m4v15 I can ask the mentors, if they're happy I'll merge.

**Comment by m4v15** [comment]
shukran jazeelan

**Comment by developess** [comment]
Yes this is great. @eliascodes @m4v15 feel free to merge!

**Pull request merged**